testdata/p4tc_samples_outputs/calculator_control_blocks.c

```diff
@@ -147,12 +148,12 @@ if (/* hdr->p4calc.isValid() */
                                     drop_packet();
                                 }
                                 break;
-                            default:
-                                return TC_ACT_SHOT;
+                            case MAINCONTROLIMPL_CALCULATE_ACT_NOACTION: 
+                                {
+                                }
+                                break;
                         }
                     } else {
-/* drop_packet() */
```

Comment:
else part removal is expected?

Reply:
Yes. else is expected to be empty to cater the case when value->action is NULL.